bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/i18n/ThingStatusInfoI18nLocalizationService.java

```diff
@@ -87,7 +87,7 @@ protected void setTranslationProvider(TranslationProvider i18nProvider) {
     }
 
     protected void unsetTranslationProvider(TranslationProvider i18nProvider) {
-        i18nProvider = null;
+        this.i18nProvider = null;
```

Comment:
@maggu2810 Just by chance I noticed on that line that the search&replace from I18nProvider->TranslationProvider was only a partial success: It is a bit nasty that all variables are still called i18nProvider :-(